src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -142,7 +143,7 @@
      [quo/summary-info
       {:type          summary-info-type
        :networks?     true
-       :values        network-values
+       :values        (send-utils/network-values-for-ui network-values)
```

Comment:
Here's one place we were using the UI strings inside network-values, because summary-info will use the string "<0.01" to determine if some UI should be shown. Not sure what the original intent of that code would be, but perhaps that code could be changed too.